src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1564,9 +1570,13 @@ def do_engine_fuzzing(self, engine_impl):
     # Do the actual fuzzing.
     for fuzzing_round in range(environment.get_value('MAX_TESTCASES', 1)):
       logs.log(f'Fuzzing round {fuzzing_round}.')
-      result, current_fuzzer_metadata, fuzzing_strategies = run_engine_fuzzer(
-          engine_impl, self.fuzz_target.binary, sync_corpus_directory,
-          self.testcase_directory)
+      try:
+        result, current_fuzzer_metadata, fuzzing_strategies = run_engine_fuzzer(
+            engine_impl, self.fuzz_target.binary, sync_corpus_directory,
+            self.testcase_directory)
+      except FuzzTargetNotFoundError:
```

Comment:
can you add a comment here explaining how we handle these exceptions? I.e. we treat missing targets as not reproducing (e.g. if they were deleted). 

 Also should we be logging this?